README.md

```diff
@@ -831,6 +831,23 @@ mrsk lock acquire -m "Doing maintanence"
 mrsk lock release
 ```
 
+## Gradual restarts
+
+When deploying to large numbers of hosts, you might prefer not to restart your services on every host at the same time.
+
+MRSK's default is to start new containers on all hosts in parallel. But you can control this by configuring `group_limit` and `group_wait`.
+
+```yaml
+service: myservice
+
+group_limit: 10
+group_wait: 30
```

Comment:
Would dial this down since folks may treat it as a default to start with. We use 2-second waits for Capistrano rolling deploys, by contrast.